lib/app.dart

```diff
@@ -5,20 +5,28 @@ import 'package:survey/gen/configs.gen.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 class App extends StatelessWidget {
+  final Widget? overrideHome;
+  const App({this.overrideHome});
+
   @override
   Widget build(BuildContext context) {
+    final routes = Configs.routes;
+    if (overrideHome != null) {
+      routes["/"] = (_) => overrideHome!;
```

Comment:
did we agree to have a const to manage all the navigation endpoint in 1 place with a better naming system rather than hardcoding in this `build` block? 🤔  I think yes right?

Reply:
This `overrideHome` is a trick to create a shortcut to a module (only for dev & test).
So not sure a constant here is necessary or where to store this constant and the name of it? 🤔 

Ex: `main-development.dart`
```dart
void main() {
  Flavour.current = FlavorValue.staging;
  bootstrap();

  run(App(overrideHome: AnyModule()));
}